app/controllers/HomeController.scala

```diff
@@ -34,28 +34,48 @@ object HomeController extends HomeController(ApplicationClient, CSRCache)
 
 class HomeController(applicationClient: ApplicationClient, cacheClient: CSRCache) extends BaseController(applicationClient, cacheClient) {
   val Withdrawer = "Candidate"
+  val baseTimeForEtrayTest = 80
 
   val present = CSRSecureAction(ActiveUserRole) { implicit request => implicit cachedData =>
     cachedData.application.map { application =>
 
       def getPhase2Test: Future[Option[Phase2TestGroupWithActiveTest]] = if (application.applicationStatus == ApplicationStatus.PHASE2_TESTS) {
         applicationClient.getPhase2TestProfile(application.applicationId).map(Some(_))
-      } else { Future.successful(None) }
+      } else {
+        Future.successful(None)
+      }
 
       def getPhase3Test: Future[Option[Phase3TestGroup]] = if (application.applicationStatus == ApplicationStatus.PHASE3_TESTS) {
         applicationClient.getPhase3TestGroup(application.applicationId).map(Some(_))
-      } else { Future.successful(None) }
+      } else {
+        Future.successful(None)
+      }
+
+      def getPhase2TimeToCompleteTest: Future[Option[Int]] = if (application.progress.phase2TestProgress.phase2TestsInvited) {
```

Comment:
It seems like this logic (and associated constant) belongs in the backend service